package.json

```diff
@@ -58,11 +61,15 @@
     "@types/redux-logger": "^3.0.6",
     "@types/redux-mock-store": "^1.0.0",
     "@types/redux-thunk": "^2.1.0",
+    "electron": "^3.0.13",
+    "electron-builder": "^20.38.3",
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.7.0",
     "foreman": "^3.0.1",
     "jest-enzyme": "^7.0.1",
+    "jquery": "^3.3.1",
```

Comment:
We're currently not using jQuery or popper.  I know they are required if you are using the bootstrap JS, but we aren't right now.

Reply:
just for note keeping sake, electron-builder won't build without these 2 (probably some strict requirement that all peer dependencies must be install)